dojox/1.11/gfx/Mover.d.ts

```diff
@@ -0,0 +1,20 @@
+declare namespace dojox {
+	namespace gfx {
+		interface Mover {
+			destroy(): void;
+			onFirstMove(): void;
+			onMouseMove(event: MouseEvent): void;
+		}
+
+		interface MoverConstructor extends dojo._base.DeclareConstructor<Mover> {
+			new(shape: dojox.gfx.shape.Shape, event: MouseEvent, host: Moveable): Mover;
```

Comment:
Does this file need a reference to shape or will TypeScript attempt to search for local .d.ts files?

```
/// <reference path="shape.d.ts" />
```

So if somebody used *only* this definition I believe they would get a missing definition error, but I'm not sure. Worth following up on.